codec/src/test/java/io/netty/handler/codec/xml/XmlFrameDecoderTest.java

```diff
@@ -44,10 +43,8 @@
     private final List<String> xmlSamples;
 
     public XmlFrameDecoderTest() throws IOException, URISyntaxException {
-        xmlSamples = Arrays.asList(
-                sample("01"), sample("02"), sample("03"),
-                sample("04"), sample("05"), sample("06")
-        );
+        xmlSamples = Arrays.asList(sample("01"), sample("02"), sample("03"),
+                sample("04"), sample("05"), sample("06"), sample("07"));
```

Comment:
Could you please keep the original formatting so that the changeset contains only the relevant changes?